source/AI.cpp

```diff
@@ -1315,7 +1344,7 @@ void AI::MoveIndependent(Ship &ship, Command &command) const
 
 
 
-void AI::MoveEscort(Ship &ship, Command &command) const
+void AI::MoveEscort(Ship &ship, Command &command, const PlayerInfo &player) const
```

Comment:
I don't think I want to package the player into MoveEscort.
But, on some level it does make sense to give MoveEscort this information, since all playerships (sans flagship) use this when they are not attacking something.

Reply:
It's the only way until there is another way to get player information.
Personally I think it makes sense to cache the player info in a variable of the AI class, but I don't have a good enough understanding of the AI class or the thread use of the game to push for that.